components/desktop/freerdp/Makefile

```diff
@@ -44,8 +44,10 @@ CMAKE_OPTIONS += -DWITH_ALSA=OFF
 CMAKE_OPTIONS += -DWITH_CAIRO=ON
 CMAKE_OPTIONS += -DWITH_PCSC=OFF
 CMAKE_OPTIONS += -DWITH_CUPS=ON
-CMAKE_OPTIONS += -DWITH_PULSE=ON
-CMAKE_OPTIONS += -DWITH_OSS=OFF
+CMAKE_OPTIONS += -DWITH_OSS=ON
+# mic input slows to a crawl if there is more than one
+# pulse client
+CMAKE_OPTIONS += -DWITH_PULSE=OFF
 CMAKE_OPTIONS += -DWITH_SSE2=ON
```

Comment:
What are the implications of switching to OSS? To my knowledge it was a decision to use only a single audio system and that was pulse.

I am willing to drop pulseaudio but we'll need something better then. Is OSS working all over? What other components would need a change and do they all work?

Reply:
> What are the implications of switching to OSS?
`xfreerdp(1)` accesses the mixer `/dev/dsp` directly which results in lower latency even with multiple apps recording or playing back to the same device. pulseaudio appears to _serialise_ access to the sound card (!) which results in massive recording or playback latency even if one app is exclusively playing back or recording audio